<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main_Window</class>
 <widget class="QMainWindow" name="Main_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1799</width>
    <height>1112</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>15</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Display_Map</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="Display_Map" name="widget" native="true"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="map_load_status">
          <property name="text">
           <string>load status:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Initial_Area</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="Initial_Area" name="widget_5" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Initial_Location</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <widget class="Initial_Location" name="widget_6" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_10">
       <attribute name="title">
        <string>Max_Extent</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="0" column="0">
         <widget class="Viewpoint_Max_Extent" name="widget_10" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_11">
       <attribute name="title">
        <string>Reference_Scale</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,4">
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string/>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>1:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="reference_scale_select">
               <item>
                <property name="text">
                 <string>500000</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>250000</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>100000</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>50000</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="button_reference_scale_syncronize">
             <property name="text">
              <string>Set Map Scale to Reference Scale</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="reference_operational_layers_list"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="Map_Reference_Scale" name="widget_11" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Spatial_Reference</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="Map_Spatial_Reference" name="widget_4" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_12">
       <attribute name="title">
        <string>Magnifier</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_9">
        <item row="0" column="0">
         <widget class="Map_Magnifier" name="widget_12" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Basemap_Change</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="Basemap_Change" name="widget_2" native="true"/>
        </item>
        <item>
         <widget class="QComboBox" name="basemap_change">
          <item>
           <property name="text">
            <string>Topographic</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Streets</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Imagery</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Oceans</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>Open_Existing_Map</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="Open_Existing_Map" name="widget_7" native="true"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QComboBox" name="map_select">
            <item>
             <property name="text">
              <string>Population Pressure</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Terrestrial Ecosystems of the World</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Geology of United States</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_map_open">
            <property name="text">
             <string>Open</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_13">
       <attribute name="title">
        <string>Mobile_Map_Package</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_10">
        <item row="0" column="0">
         <widget class="Mobile_Map_Package" name="widget_13" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_14">
       <attribute name="title">
        <string>Geo_Package</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_11">
        <item row="0" column="0">
         <widget class="Geo_Package" name="widget_14" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_15">
       <attribute name="title">
        <string>Identify_Layers</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_12">
        <item row="0" column="0">
         <widget class="Identify_Layers" name="widget_15" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_8">
       <attribute name="title">
        <string>Bookmark_Manage</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,4">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string/>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QListWidget" name="list_bookmark"/>
           </item>
           <item>
            <widget class="QPushButton" name="button_create_bookmark">
             <property name="text">
              <string>Create Bookmark Here</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="Bookmark_Manage" name="widget_8" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Viewpoint_Change</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="1">
         <widget class="QComboBox" name="viewpoint_change">
          <item>
           <property name="text">
            <string>Center</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Center and scale</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Geometry</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Geometry and padding</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Rotation</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Scale: 1:5,000,000</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Scale: 1:10,000,000</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Animation</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="Viewpoint_Change" name="widget_3" native="true"/>
        </item>
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Control</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="2" column="0" colspan="2">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Hawaii: -157.564, 20.677</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="9" column="0" colspan="2">
            <widget class="QPushButton" name="viewpoint_change_scale">
             <property name="text">
              <string>Scale</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="2">
            <widget class="QPushButton" name="viewpoint_change_center_scale">
             <property name="text">
              <string>Center_scale</string>
             </property>
            </widget>
           </item>
           <item row="8" column="0" colspan="2">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>1:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="viewpoint_scale_value">
               <property name="text">
                <string>5000000.0</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="7" column="0" colspan="2">
            <widget class="QPushButton" name="viewpoint_change_rotate">
             <property name="text">
              <string>Rotate</string>
             </property>
            </widget>
           </item>
           <item row="11" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>duration</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="viewpoint_animate_duration">
               <property name="text">
                <string>4</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="5" column="0" colspan="2">
            <widget class="QPushButton" name="viewpoint_change_geometry">
             <property name="text">
              <string>Geometry</string>
             </property>
            </widget>
           </item>
           <item row="12" column="0" colspan="2">
            <widget class="QPushButton" name="viewpoint_change_scale_animationCurve">
             <property name="text">
              <string>Scale_AnimationCurve</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0" colspan="2">
            <widget class="QPushButton" name="viewpoint_change_geometry_padding">
             <property name="text">
              <string>Geometry_Padding</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QPushButton" name="viewpoint_change_center">
             <property name="text">
              <string>Center</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0" colspan="2">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Beijing Area: 116.380, 39.920, 116.400, 39.940</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="11" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>animatetype</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="viewpoint_animate_type">
               <item>
                <property name="text">
                 <string>EaseInOutCubic</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item row="0" column="0" colspan="2">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Esri_Headquarters: -117.195681, 34.056218</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="10" column="0" colspan="2">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>View Point Spring: Envelope(-12338668, 5546908, -12338247, 5547223)</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_9">
       <attribute name="title">
        <string>Basemap_Language</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="Basemap_Language" name="widget_9" native="true"/>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>GroupBox</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QComboBox" name="basemap_language_select">
             <item>
              <property name="text">
               <string>none</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Bulgarian</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Greek</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Turkish</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="basemap_language_is_global">
             <property name="text">
              <string>Is Global</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_16">
       <attribute name="title">
        <string>Location_History</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="Location_History" name="widget_16" native="true"/>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_5">
          <property name="title">
           <string/>
          </property>
          <layout class="QGridLayout" name="gridLayout_13">
           <item row="0" column="0">
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>Start Tracking</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1799</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Display_Map</class>
   <extends>QWidget</extends>
   <header location="global">Demo/Display_Map.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Basemap_Change</class>
   <extends>QWidget</extends>
   <header location="global">Demo/Basemap_Change.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Viewpoint_Change</class>
   <extends>QWidget</extends>
   <header location="global">Demo/Viewpoint_Change.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Map_Spatial_Reference</class>
   <extends>QWidget</extends>
   <header location="global">Demo/Map_Spatial_Reference.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Initial_Area</class>
   <extends>QWidget</extends>
   <header location="global">Demo/Initial_Area.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Initial_Location</class>
   <extends>QWidget</extends>
   <header location="global">Demo/Initial_Location.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Open_Existing_Map</class>
   <extends>QWidget</extends>
   <header location="global">Demo/Open_Existing_Map.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Bookmark_Manage</class>
   <extends>QWidget</extends>
   <header location="global">Demo/Bookmark_Manage.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Basemap_Language</class>
   <extends>QWidget</extends>
   <header location="global">Demo/Basemap_Language.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Viewpoint_Max_Extent</class>
   <extends>QWidget</extends>
   <header location="global">Demo/Viewpoint_Max_Extent.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Map_Reference_Scale</class>
   <extends>QWidget</extends>
   <header location="global">Demo/Map_Reference_Scale.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Map_Magnifier</class>
   <extends>QWidget</extends>
   <header location="global">Demo/Map_Magnifier.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Mobile_Map_Package</class>
   <extends>QWidget</extends>
   <header location="global">Demo/Mobile_Map_Package.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Geo_Package</class>
   <extends>QWidget</extends>
   <header location="global">Demo/Geo_Package.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Identify_Layers</class>
   <extends>QWidget</extends>
   <header location="global">Demo/Identify_Layers.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Location_History</class>
   <extends>QWidget</extends>
   <header location="global">Demo/Location_History.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
